lib/state/appActions.js

```diff
@@ -247,6 +250,12 @@ appActions.load = function(cb) {
 
 appActions.login = function(credentials, options, cb) {
   var self = this;
+
+  var err = appActions._assertUploaderVersion();
```

Comment:
so I am trying to assert that the version of the uploader they are using is allowed to upload to the platform. If it isn't a required version then they are prompted to update before going any further